tools/static-code-analysis/spotbugs/suppressions.xml

```diff
@@ -24,6 +24,14 @@
     <Class name="~.*Util"/>
     <Bug pattern="SLF4J_LOGGER_SHOULD_BE_NON_STATIC"/>
   </Match>
+  <Match>
+    <Package name="org.openhab.binding.astro.internal.calc"/>
+    <Bug pattern="SLF4J_LOGGER_SHOULD_BE_NON_STATIC"/>
+  </Match>
+  <Match>
+    <Package name="org.openhab.binding.astro.internal.job"/>
+    <Bug pattern="SLF4J_LOGGER_SHOULD_BE_NON_STATIC"/>
+  </Match>
```

Comment:
Not sure if i like this. before we know it this becomes a precedent for (many) others and as you know we like to keep things consistent.
Did you consider other options?

Reply:
It also comes with a lot of other annotations, including null-annotations, that I don't think would fit that nicely within OH: https://github.com/spotbugs/spotbugs/tree/d27531c1a282207e8a098d1e094bd11cea826d8b/spotbugs-annotations/src/main/java/edu/umd/cs/findbugs/annotations

So, I think it's pretty much down to global suppression or renaming classes... What do you prefer?

Reply:
I think it's a good practice to not use static loggers, so that the logger is recycled with the instance, but that only works for non-static code. So, it's a "good rule", but it's too rigid IMO.